[BUG]: Various Warnings related to Package

## Warning related to configuration

The package was inspired from [k8ssandra-operator](https://docs.k8ssandra.io/install/local/) and it involves passing values from a values.yaml, it'd be really useful if one can patch the warning related to the maps.



## Issue
Currently there seems to be a [misconfiguration](https://github.com/dokc/Helm-Charts/blob/4dc1c39fb0739649809eb3da72d8a646a6edc549/values.yaml#L69-L81) where the `garbage-collector`and `full-query-logging(fql)` is not configured well. One needs to go through and patch the warning messages.
The warning messages look like this:

## Console
```go
coalesce.go:220: warning: 
cannot overwrite table with non table for k8s-lightweight.cassandra.datacenters (map[audit_logging:map[] client_backpressure:map[] fql:map[] gc:map[cms:map[] g1:map[]] heap:map[] name:dc1 racks:map[affinityLabels:map[] name:default] size:1])

coalesce.go:220: warning: 
cannot overwrite table with non table for k8s-lightweight.cassandra.datacenters (map[audit_logging:map[] client_backpressure:map[] fql:map[] gc:map[cms:map[] g1:map[]] heap:map[] name:dc1 racks:map[affinityLabels:map[] name:default] size:1])

```

--- a/.github/workflows/sanity-check.yaml
+++ b/.github/workflows/sanity-check.yaml
@@ -1,0 +1,17 @@
+name: Sanity check
+on: [pull_request]
+
+jobs:
+  commits_check_job:
+    runs-on: ubuntu-latest
+    name: Commits Check
+    steps:
+      - name: Get PR Commits
+        id: "get-pr-commits"
+        uses: tim-actions/get-pr-commits@master
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: DCO Check
+        uses: tim-actions/dco@master
+        with:
+          commits: ${{ steps.get-pr-commits.outputs.commits }}

--- a/Chart.yaml
+++ b/Chart.yaml
@@ -3,18 +3,18 @@ apiVersion: v2
 dependencies:
   - condition: cass-operator.enabled
     name: cass-operator
-    repository: file://../cass-operator
+    repository: file://./charts/cass-operator
     version: 0.32.0
   - condition: reaper-operator.enabled
     name: reaper-operator
-    repository: file://../reaper-operator
+    repository: file://./charts/reaper-operator
     version: 0.32.3
   - condition: medusa.enabled
     name: medusa-operator
-    repository: file://../medusa-operator
+    repository: file://./charts/medusa-operator
     version: 0.32.0
   - name: k8ssandra-common
-    repository: file://../k8ssandra-common
+    repository: file://./charts/k8ssandra-common
     version: 0.28.4
   - condition: kube-prometheus-stack.enabled
     name: kube-prometheus-stack

--- a/Chart.yaml
+++ b/Chart.yaml
@@ -1,4 +1,5 @@
-apiVersion: v1
+apiVersion: v2
+
 dependencies:
   - condition: cass-operator.enabled
     name: cass-operator
@@ -19,13 +20,13 @@ dependencies:
     name: kube-prometheus-stack
     repository: https://prometheus-community.github.io/helm-charts
     version: 20.0.1
+
 description: "Provisions a simple, minimalistic K8ssandra stack. This includes Apache Cassandra (Database), Stargate (REST API), Prometheus (Metrics Middleware), and Grafana. (Data Visualization) "
+
 maintainers:
-  - email: "Abhijith_Ganesh@outlook.com"
-    name: Abhijith Ganesh
-    url: https://github.com/AbhijithGanesh/dokc-helm-chart
-name: k8ssandra
-sources:
-  - https://github.com/AbhijithGanesh/dokc-helm-chart
+  - name: Data on Kuberentes Community
+    url: https://dok.community
+
+name: k8s-lightweight
 type: application
-version: 1.4.1
+version: 0.1.0

--- a/README.md
+++ b/README.md
@@ -1,2 +1,103 @@
 # dokc-helm-chart
 Custom K8ssandra operator for Data on Kuberenetes' how-to-dok project.
+
+## What is the custom k8ssandra-lightweight operator ?
+
+This the kubernetes-operator which contains the following packages:
+    1. Apache Cassandra
+    2. Stargate 
+    3. Prometheus
+    4. Grafana
+
+This package a lightweight package
+
+## Installation and requirement
+
+### Installation
+To install this package, you can add this to your helm repositories by 
+
+- `helm repo add dokc https://dokc.github.io/Helm-Charts/`
+
+Once added, update the repository by running
+
+- `helm repo update`
+
+To check the repositories in your local setup, check if the `dokc` repo is added by running
+
+- `helm repo list`
+
+It must show something like this
+```
+NAME                    URL
+cassandra               https://helm.k8ssandra.io/stable
+traefik                 https://helm.traefik.io/traefik
+minio                   https://helm.min.io
+prometheus-community    https://prometheus-community.github.io/helm-charts
+```
+
+Once the repository is listed, you can run your k8ssandra cluster, but this would require a YAML file which would like this
+_Replace: < > with values_
+
+```yaml
+ cassandra:
+  version: < Cassandra Version Number, preferred version = 4.0.1 >  
+  cassandraLibDirVolume:
+    storageClass: local-path
+    size: 5Gi
+  allowMultipleNodesPerWorker: true
+  heap:
+   size: 1G
+   newGenSize: 1G
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 2Gi
+    limits:
+      cpu: 1000m
+      memory: 2Gi
+  datacenters:
+  - name: < Data Center Name >
+    size: < Size >
+    racks:
+    - name: < Rack Name >
+    
+kube-prometheus-stack:
+  grafana:
+    adminUser: < Grafana Admin User >
+    adminPassword: < Grafana Password >
+    
+    
+stargate:
+  enabled: < Boolean >
+  replicas: 1
+  heapMB: 256
+  cpuReqMillicores: 200
+  cpuLimMillicores: 1000
+  
+```
+
+To run the cluster, execute the following command: helm install -f <filename>.yaml <Instance Name (or) Cluster Name> dokc/k8s-lightweight
+
+** The process should roughly take 4 minutes , wait for the pods to start up **
+
+To check the status of the pods run:
+
+`kubectl get pods`
+
+```
+NAME                                                   READY   STATUS    RESTARTS   AGE
+prometheus-test-dep-kube-prometheus-s-prometheus-0     2/2     Running   0          2m43s
+test-dep-cass-operator-6c75d6d684-vh2ts                1/1     Running   0          2m45s
+test-dep-datacenter1-default-sts-0                     2/2     Running   0          2m28s
+test-dep-datacenter1-stargate-699f64f646-w85kw         1/1     Running   0          2m45s
+test-dep-grafana-57fd996d9-c58tf                       2/2     Running   0          2m45s
+test-dep-kube-prometheus-s-operator-767fc8984c-9rlmj   1/1     Running   0          2m45s
+```
+
+Disclaimer: "It may work within 4 minutes, but it is advisable to wait for 4 minutes"
+
+
+### Requirements:
+ - CPU -> 4 Cores
+ - Memory -> 8128M
+

--- a/artifacthub-repo.yml
+++ b/artifacthub-repo.yml
@@ -1,4 +1,3 @@
 owners: 
-  - name: AbhijithGanesh
-    email: Abhijith_Ganesh@outlook.com
+  - name: Data On Kuberenetes Community
   

--- a/index.yaml
+++ b/index.yaml
@@ -1,9 +1,12 @@
-apiVersion: v1
+apiVersion: v2
 entries:
-  dokc-chart:
-    - apiVersion: v1
+  k8s-lightweight:
+    - apiVersion: v2
       name: "Data on Kuberenetes K8ssandra Chart"
-      home: https://github.com/AbhijithGanesh/dokc-helm-chart
-      description: "A chart for making k8ssandra simple. You can read about how to use them here in this ."
-
+      home: https://github.com/dokc/Helm-Charts.git
+      description: "A lightweight chart for K8ssandra operator."
+      maintainers:
+          name: Data on Kuberenetes Community
+          community: https://dokcommunity.slack.com/join/shared_invite/zt-10v7uncvp-jNFwulsVWvUO0SKMDTjwAw#/shared-invite/email
+      
 generated: "2022-02-01T16:32:21.7893764+05:30"

--- a/values.yaml
+++ b/values.yaml
@@ -27,7 +27,6 @@ cassandra:
       tag: 4.14.1
       pullPolicy: IfNotPresent
   jmxCredentialsConfig:
-    securityContext: {}
     image:
       registry: docker.io
       repository: busybox
@@ -66,19 +65,6 @@ cassandra:
     g1: {}
   resources: {}
   tolerations: []
-  datacenters:
-    name: dc1
-    size: 1
-    racks:
-      name: default
-      affinityLabels: {}
-    fql: {}
-    audit_logging: {}
-    client_backpressure: {}
-    heap: {}
-    gc:
-      cms: {}
-      g1: {}
   ingress:
     enabled: false
     method: traefik
@@ -146,11 +132,8 @@ stargate:
   tolerations: []
 
 reaper:
-  securityContext: {}
   schemaInitContainerConfig:
-    securityContext: {}
   configInitContainerConfig:
-    securityContext: {}
   podSecurityContext: {}
   autoschedule: false
   autoschedule_properties: {}
@@ -178,9 +161,7 @@ reaper:
 
 medusa:
   enabled: false
-  securityContext: {}
   restoreInitContainerConfig:
-    securityContext: {}
   image:
     registry: docker.io
     repository: k8ssandra/medusa

